### PR TITLE
Add Roadmap page

### PR DESCRIPTION
**Pull request by scholzj**
I though it would be useful to have a roadmap on the Strimzi website. Please review.

### CODE REVIEW

File: roadmap/index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+All Strimzi deployments are currently running without any resource requests and limits (see 
+[Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/) 
+for more details). A possibility to configure resource limits and requests should be added to the Cluster Controller 
+component.
+
+## Add support for Kubernetes node selectors
+
+In order to achieve good performnce, it might be needed to schedule Kafka pods to specific nodes. For example because 
+they contain special storage hardware or just to run them on dedicated nodes to make sure that KAfka doesn't interfere
```
Comment (ppatierno) [inline roadmap/index.md]: typo "KAfka"

File: roadmap/index.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+In order to achieve good performnce, it might be needed to schedule Kafka pods to specific nodes. For example because 
+they contain special storage hardware or just to run them on dedicated nodes to make sure that KAfka doesn't interfere 
+with any other applications. To achiave this, it should be possible to configure node selectors for the Strimzi 
+deployments.
+
+## Custom Resource Definitions support
+
+Cluster and Topic Controllers should have support for [Custom Resource Definitions (CRDs)](https://kubernetes.io/docs/concepts/api-extension/custom-resources/).
+
+## Service broker suport
```
Comment (ppatierno) [inline roadmap/index.md]: typo "suport"

**Comment by scholzj** [comment]
@ppatierno I think that the top of the list should be fairly according to our priorities. The more you get to the bottom, the less clear the prioritization is. I think that is fairly normal for such a roadmap.

**Comment by ppatierno** [comment]
@scholzj ok got it !

**Pull request merged**